No setup_requires pytest-runner, it is only for the tests.

diff -Nrup a/setup.py b/setup.py
--- a/setup.py	2024-01-25 03:57:05.000000000 +0100
+++ b/setup.py	2026-10-22 17:22:14.747116952 +0200
@@ -191,9 +191,6 @@ setuptools.setup(
     cmdclass={
         'test': PyTest,
     },
-    setup_requires=[
-        'pytest-runner',
-    ],
     scripts=[
         'scripts/stargate',
     ],
